Gateway/Request/APMBuilder.php

```diff
@@ -82,6 +88,12 @@ public function build(array $buildSubject)
                     self::SOURCE_INSTALLMENT_TERMS => $method->getAdditionalInformation(InstallmentDataAssignObserver::TERMS),
                 ];
                 break;
+            case Truemoney::CODE:
```

Comment:
Again, this implementation feels like it should be separate classes for each APM

Reply:
@jonrandy could you present more info about why you think it should be done in separated files each one?

Reply:
The comment directly relates to my previous explanation and comments by @guzzilar 

The reasoning is the same

Reply:
Ok, I don't see any real problems it would solve now, yes.. it would nice to have, but definitely not in this PR, this potential refactoring concerns all payment methods.